samples/tacos/src/main/kotlin/com/slack/circuit/tacos/ui/theme/Color.kt

```diff
@@ -16,8 +16,8 @@ val md_theme_light_secondaryContainer = Color(0xFFE8DEF8)
 val md_theme_light_onSecondaryContainer = Color(0xFF1E192B)
 val md_theme_light_tertiary = Color(0xFF7E5260)
 val md_theme_light_onTertiary = Color(0xFFFFFFFF)
-val md_theme_light_tertiaryContainer = Color(0xFF7E5260)
-val md_theme_light_onTertiaryContainer = Color(0xFFBA1A1A)
+val md_theme_light_tertiaryContainer = Color(0xFFFFD9E3)
+val md_theme_light_onTertiaryContainer = Color(0xFF31101D)
```

Comment:
When I imported the generated colour scheme I changed these two for use by the red bottom bar. While this worked it would have negative affects on other Material 3 components that use those named colours.

Reverting back to the generated defaults and defining some new colours down below for this specific use case